### ISSUE TITLE
Failing to create dr S3 bucket in the desired region

### ISSUE THREAD

**Issue by g-iannelli** [sighupio/furyctl#261]
```shell

cat barebone.yml|  yq -e .spec.distribution.modules.dr.velero.eks.region
us-east-1
❯ cat barebone.yml|  yq -e .spec.distribution.modules.dr.velero.eks.bucketName
furyctl-eks-barebone-wezmvdhr
❯ aws s3api get-bucket-location --bucket furyctl-eks-barebone-wezmvdhr | tee
{
    "LocationConstraint": "eu-west-1"
}
```

**Comment by g-iannelli** [comment]
Duplicated and solved by [fury-distribution#58](https://github.com/sighupio/fury-distribution/pull/58/files#diff-7c6ea445857e221583924f3a130f7023523bfd92f6281f4091e9616045a7a26b)

**Comment by omissis** [comment]
closing as duplicate
